neo/test/iotest/test_nixio_fr.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""
+Tests of neo.io.nixio_fr
+"""
+from __future__ import absolute_import
+import numpy as np
+import unittest
+from neo.io.nixio_fr import NixIO as NixIOfr
+import quantities as pq
+from neo.io.nixio import NixIO
+from neo.test.iotest.common_io_test import BaseTestIO
+from neo.test.iotest.tools import get_test_file_full_path
+try:
+    import nixio as nix
+
+    HAVE_NIX = True
+except ImportError:
+    HAVE_NIX = False
+
+
+
+
+@unittest.skipUnless(HAVE_NIX, "Requires NIX")
+class TestNixfr(BaseTestIO, unittest.TestCase, ):
+    ioclass = NixIOfr
+
+    files_to_test = ['nixio_fr.nix']
+
+    files_to_download = ['nixio_fr.nix']
+
+    def setUp(self):
+        super(TestNixfr, self).setUp()
+        self.testfilename = 'nixio_fr.nix'
```

Comment:
this line needs to be changed to get the correct file path:
```suggestion
        self.testfilename = 'self.get_filename_path('nixio_fr.nix')
```